.github/workflows/release.yml

```diff
@@ -1,9 +1,27 @@
-name: OCI Image Publish
+name: Release
 on:
   workflow_dispatch: null
   release:
     types: [ published ]
 jobs:
+  notify:
```

Comment:
Can we move this step after all other published jobs and notify the channel if it succeeded?

Reply:
Thanks! Do you think `needs: pypi-release` is sufficient in this case? I think when docker publish failed we can manually re-release it.